web/src/containers/ActivityDetailAll.js

```diff
@@ -22,25 +22,36 @@ const activityTitle = (a, i) => {
   return title;
 };
 
-const ActivityDetailAll = ({ aId, expanded, title, toggleSection }) => (
-  <Collapsible
-    id={`activity-${aId}`}
-    title={title}
-    bgColor="darken-1"
-    open={expanded}
-    onChange={() => toggleSection(aId)}
-  >
-    <ActivityDetailDescription aId={aId} />
-    <ActivityDetailGoals aId={aId} />
-    <ActivityDetailSchedule aId={aId} />
-    <ActivityDetailStatePersonnel aId={aId} />
-    <ActivityDetailContractorResources aId={aId} />
-    <ActivityDetailExpenses aId={aId} />
-    <ActivityDetailCostAllocate aId={aId} />
-    <ActivityDetailStandardsAndConditions aId={aId} />
-    <DeleteActivity aId={aId} />
-  </Collapsible>
-);
+class ActivityDetailAll extends Component {
+  handleChange = id => () => {
```

Comment:
for performance, better to memoize this method so it doesn't get recreated on each render